static/js/util/withTracker.js

```diff
@@ -3,18 +3,19 @@
 
 // From https://github.com/ReactTraining/react-router/issues/4278#issuecomment-299692502
 import React from "react"
-import ga from "react-ga"
+import ReactGA from "react-ga"
 
 const withTracker = (WrappedComponent: Class<React.Component<*, *>>) => {
   const debug = SETTINGS.reactGaDebug === "true"
+  const test = !!SETTINGS.isTest
```

Comment:
is this just to make sure it's in test mode in the tests? if so I think it might be a little cleaner to check if a node-only global is defined, for instance `process`, rather than adding a new `isText` boolean to our `SETTINGS` global.

Reply:
and maybe additionally check the `process.env` is a value we expect (think it's `'development'` normally)